nvflare/apis/utils/fl_context_utils.py

```diff
@@ -30,7 +30,9 @@ def get_serializable_data(fl_ctx: FLContext):
                 fobs.dumps(v)
                 new_fl_ctx.props[k] = v
             except BaseException as e:
-                logger.warning(generate_log_message(fl_ctx, f"Object is not serializable (discarded): {k} - {v}: {e}"))
+                logger.warning(
+                    generate_log_message(fl_ctx, f"Object is not serializable (discarded): {k} - {type(v)}: {e}")
```

Comment:
Can we move this change to another PR thanks

Reply:
Sure. Restored the original file.